saleor/graphql/order/mutations/draft_orders.py

```diff
@@ -314,6 +314,14 @@ def save(cls, info, instance, cleaned_input):
                 "Unable to calculate taxes - %s" % str(tax_error),
                 code=OrderErrorCode.TAX_ERROR.value,
             )
+        except Exception as e:
+            raise ValidationError(
+                {
+                    "shipping_method": ValidationError(
```

Comment:
Is it safe enough that we assume, that `DraftOrderCreate` mutation, when raises a generic exception, means - inccorect shipping_method? Assuming that we will change anything, or break something inside this `try,except`, then DraftOrderCreate will raise incorrect exception.